src/platform/forms-system/src/js/routing.js

```diff
@@ -5,7 +5,7 @@ import { getActiveExpandedPages } from './helpers';
  * Returns the page list without conditional pages that have not satisfied
  * their dependencies and therefore should be skipped.
  */
-function getEligiblePages(pageList, data, pathname) {
+export function getEligiblePages(pageList, data, pathname) {
```

Comment:
I'm not aware of the reasoning behind why this function wasn't previously exposed, but there's a design pattern becoming more and more common in the frontend - where a form displays read-only veteran information for logged in users, and schemafields for logged out users - that benefits from having this function directly accessible. Calling this directly from the introduction page allows modification to the page list passed in to `<SaveInProgressIntro />`. This is helpful, especially when the first page in a form contains a `depends` attribute for toggling it on/off.

Reply:
Ohhh, interesting. :thinking: 

I can't think of a reason this shouldn't be exported myself. Thanks for the explanation! :raised_hands: